src/style.css

```diff
@@ -108,6 +112,13 @@ main {
   margin: 40px 0;
 }
 
+.comment-formr button {
```

Comment:
Kindly, use proper name for classes, it will help you in future when you visit again.